src/Symfony/Component/Yaml/Parser.php

```diff
@@ -381,6 +381,11 @@ private function doParse(string $value, int $flags)
                         if (0 === $this->offset && !$deprecatedUsage && isset($line[0]) && ' ' === $line[0]) {
                             throw new ParseException('Unable to parse.', $this->getRealCurrentLineNb() + 1, $this->currentLine, $this->filename);
                         }
+
+                        if ($this->offset >= 1 && strpos($line, ': ')) {
```

Comment:
Done. For the offset >= 1 condition, as far as I remember, I added it because it broke some tests. As of now, without this condition, the test passes.